docs/getting-started/tutorials/microservice-with-metrics.mdx

```diff
@@ -0,0 +1,208 @@
+---
+id: microservice-with-metrics
+title: Add metrics to your custom Microservice
+sidebar_label: Add metrics to your custom Microservice
```

Comment:
We can remove "custom" to the sidebar label.

Reply:
Hi @akelity sure! I've updated the sidebar only